services-directions-models/src/main/java/com/mapbox/api/directions/v5/models/Incident.java

```diff
@@ -0,0 +1,416 @@
+
+  /**
+   * One of incident types.
+   *
+   * @see IncidentType
+   */
+  @Nullable
+  @IncidentType
+  public abstract String type();
+
+  /**
+   * <b>True</b> if road is closed and no possibility to pass through there. <b>False</b>
+   * otherwise.
+   */
+  @Nullable
+  public abstract Boolean closed();
+
+  /**
+   * Quantitative descriptor of congestion.
+   */
+  @Nullable
+  public abstract Congestion congestion();
+
+  /**
+   * Human-readable description of the incident suitable for displaying to the users.
+   */
+  @Nullable
+  public abstract String description();
+
+  /**
+   * Human-readable long description of the incident suitable for displaying to the users.
+   */
+  @Nullable
+  @SerializedName("long_description")
+  public abstract String longDescription();
+
+  /**
+   * Severity level of incident.
+   *
+   * @see ImpactType
+   */
+  @Nullable
+  @ImpactType
+  public abstract String impact();
+
+  /**
+   * Sub-type of the incident.
+   */
+  @Nullable
+  @SerializedName("sub_type")
+  public abstract String subType();
+
+  /**
+   * Sub-type-specific description.
+   */
+  @Nullable
+  @SerializedName("sub_type_description")
+  public abstract String subTypeDescription();
+
+  /**
+   * AlertC codes.
+   *
+   * @see <a href="https://www.iso.org/standard/59231.html">AlertC</a>
+   */
+  @Nullable
+  @SerializedName("alertc_codes")
+  public abstract List<Integer> alertcCodes();
+
+  /**
+   * Incident's geometry index start point.
+   */
+  @Nullable
+  @SerializedName("geometry_index_start")
+  public abstract Integer geometryIndexStart();
+
+  /**
+   * Incident's geometry index end point.
+   */
+  @Nullable
+  @SerializedName("geometry_index_end")
+  public abstract Integer geometryIndexEnd();
+
+  /**
+   * Time the incident was created/updated in ISO8601 format. Not the same
+   * {@link #startTime()}/{@link #endTime()}, incident can be created/updated before the incident.
+   */
+  @Nullable
+  @SerializedName("creation_time")
+  public abstract String creationTime();
+
+  /**
+   * Start time of the incident in ISO8601 format.
+   */
+  @Nullable
+  @SerializedName("start_time")
+  public abstract String startTime();
+
+  /**
+   * End time of the incident in ISO8601 format.
+   */
+  @Nullable
+  @SerializedName("end_time")
+  public abstract String endTime();
+
+  /**
+   * Create a new instance of this class by using the {@link Incident.Builder} class.
+   *
+   * @return this classes {@link Incident.Builder} for creating a new instance
+   */
+  public static Builder builder() {
+    return new AutoValue_Incident.Builder();
+  }
+
+  /**
+   * Convert the current {@link Incident} to its builder holding the currently assigned
+   * values. This allows you to modify a single property and then rebuild the object resulting in
+   * an updated and modified {@link Incident}.
+   *
+   * @return a {@link Builder} with the same values set to match the ones defined in this
+   *   {@link Incident}
```

Comment:
NIT should we add this in L#266? It seems this is wrongly formatted and could be avoided.

Reply:
no, it's more than 100 characters then, but I formatted L#267 👍